core/retries/src/main/java/software/amazon/awssdk/retries/DefaultRetryStrategy.java

```diff
@@ -44,13 +44,33 @@ private DefaultRetryStrategy() {
     public static StandardRetryStrategy.Builder standardStrategyBuilder() {
         return StandardRetryStrategy.builder()
                                     .maxAttempts(Standard.MAX_ATTEMPTS)
-                                    .backoffStrategy(BackoffStrategy.exponentialDelay(Standard.BASE_DELAY, Standard.MAX_BACKOFF))
-                                    .circuitBreakerEnabled(true);
+                                    .backoffStrategy(BackoffStrategy.exponentialDelay(Standard.BASE_DELAY, Standard.MAX_BACKOFF));
```

Comment:
This is a container class to give access to the different builders, and not an implementing class itself, the original design named this `RetryStrategies` but that name cannot be used since there's a validation step that complains about the name being plural, since we already have the `RetryStrategy` name taken I went with this one. Feel free to suggest another one.